department-of-veterans-affairs/caseflow
# Justin/CASEFLOW-4599-emo-unassigned-tab

Resolves [CASEFLOW-4599](https://vajira.max.gov/browse/CASEFLOW-4599)

### Description
Adds an Unassigned tab for the EMO Queue and creates spec tests to make sure the tab displays the correct appeals

### Acceptance Criteria
- [ ] When cases are assigned to the EMO queue, they show up on the Unassigned tab
- [ ] When an appeal is sent back to EMO, from an RPO organization, the appeal is removed from the Assigned tab and is shown on the Unassigned tab
- [ ] When an appeal is sent back to EMO, from BVA intake, the appeal is removed from the Assigned tab and is shown on the Unassigned tab

### Code Files Changed
- app/models/organizations/education_emo.rb
- app/models/queue_tabs/education_emo_unassigned_tasks_tab.rb
- client/COPY.json
- client/constants/QUEUE_CONFIG.json
- spec/factories/organization.rb
- spec/models/queue_tabs/education_emo_unassigned_tasks_tab.rb
#### Screenshot of code changes
![github com_department-of-veterans-affairs_caseflow_compare_feature_CASEFLOW-2738 Justin_CASEFLOW-4599-emo-unassigned-tab_expand=1](https://user-images.githubusercontent.com/95875751/163493204-6e962c94-7d7c-4af5-ba96-2d29cf0c9805.png)

### Testing Plan
1. Log in as BVA Intake user
2. Create an Appeal to send to education predocket
3. Log in as EMO user
4. Switch to EMO tabs view
5. Appeal should show in the Unassigned tab

### User Facing Changes
![image](https://user-images.githubusercontent.com/95875751/163491094-25420455-8c12-4481-9b7a-1ed3a018ce05.png)
[Link to Designs](https://bahdigital.invisionapp.com/share/QMIAIA7TANS#/screens/296070388?browse)

### Code Documentation Updates
